src/Util/StringUtil.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the RollerworksSearch package.
+ *
+ * (c) Sebastiaan Stok <s.stok@rollerscapes.net>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Rollerworks\Component\Search\Util;
+
+/**
+ * @author Issei Murasawa <issei.m7@gmail.com>
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ */
+final class StringUtil
```

Comment:
This was actually not planned for this pull-request, but we do need it in the future 😅